<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>686</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Selection Window</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/Home_logo_square.jpg</normaloff>../icons/Home_logo_square.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="1" alignment="Qt::AlignHCenter">
     <widget class="QRadioButton" name="rightSideButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(68, 85, 95);</string>
      </property>
      <property name="text">
       <string>Right </string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="loadPhotoButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>325</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(68, 85, 95);</string>
      </property>
      <property name="text">
       <string>Click to Upload Photograph</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(68, 85, 95);</string>
      </property>
      <property name="text">
       <string>Please Select the Reference Side:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="nextButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(68, 85, 95);</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="ImageDisplay" name="photoDisplay">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>250</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>250</height>
       </size>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2" alignment="Qt::AlignHCenter">
     <widget class="QRadioButton" name="leftSideButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(68, 85, 95);</string>
      </property>
      <property name="text">
       <string>Left </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QComboBox" name="ExpressionComboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="expressionsLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Expressions</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="previousButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(68, 85, 95);</string>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageDisplay</class>
   <extends>QGraphicsView</extends>
   <header>ImageDisplay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
